specification/dartLangSpec.tex

```diff
@@ -13654,16 +13657,30 @@ \subsubsection{Ordinary Invocation}
 
 \LMHash{}%
 It is a compile-time error to invoke any of the methods of class \code{Object}
-on a constant type literal that is immediately followed by the token `.'\,.
+on a type literal that is immediately followed by the token `.' (a period).
 
 \rationale{%
-The reason for the latter is that this syntax is reserved for
-invocation of static methods.
+The reason for this rule is that this syntax is reserved for
+invocation of static members.
+Invocation of a static member of a class, mixin, or extension
+uses said entity as a \emph{namespace},
+not as an actual class, mixin, or extension.
+In particular, the syntactic receiver is not evaluated
+to an object---that would not even be possible for an extension.%
+}
```

Comment:
Not sure what I'm expected to learn from this.
Especially since "this syntax" is just syntax of the form `id.id` which can be used for almost anything. It's not the syntax which is important, but perhaps the "construction" of a member access on a type literal.